aws-api-payload.json

```diff
@@ -171,7 +171,7 @@
                 "Properties": {
                     "hive.metastore.schema.verification": "false",
                     "hive.metastore.client.factory.class": "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory",
-                    "hive.metastore.warehouse.dir": "s3://$ADG_S3_PUBLISH_BUCKET/hive/external"
+                    "hive.metastore.warehouse.dir": "s3://$S3_PUBLISH_BUCKET/hive/external"
```

Comment:
Should be `s3://$S3_PUBLISH_BUCKET/analytical-dataset/hive/external`

Reply:
done